devel/ci/Dockerfile-header

```diff
@@ -1,7 +1,9 @@
 FROM registry.fedoraproject.org/fedora:FEDORA_RELEASE
 MAINTAINER "Randy Barlow" <bowlofeggs@fedoraproject.org>
 
-RUN dnf upgrade -y
+# The echo works around https://bugzilla.redhat.com/show_bug.cgi?id=1483553 and any other future dnf
+# upgrade bugs.
+RUN dnf upgrade -y || echo "We are not trying to test dnf upgrade, so ignoring dnf failure."
```

Comment:
this should be already fixed...

Reply:
@ignatenkobrain it was fixed in the repo, but not in the f27 container yet.